code/modules/research/nodes/combat.dm

```diff
@@ -113,7 +113,7 @@
 
 /datum/technology/adv_exotic_weaponry
 	name = "Advanced Experemental Weaponry"
-	desc = "Specisl weapon system using plasma as catalyst. Special weapon core prototipe that deal DNA damage to target."
+	desc = "Advanced prototype weaponry for guranteed takedown."
```

Comment:
We need description of technologies, not of what will do weapon. In desc must be something that describe method of tech